app/controllers/providers/transactions_controller.rb

```diff
@@ -41,6 +42,14 @@ def bank_transactions
         .order(happened_at: :desc, description: :desc)
     end
 
+    def disregarded_state_benefits_list
+      benefits_list = CFE::ObtainStateBenefitTypesService.call
```

Comment:
Since this "CFE::ObtainStateBenefitTypesService.call" does a HTTP request to CFE, we should rescue its errors and use Raven.capture_exception.